strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NugetArtifactGenerator.java

```diff
@@ -146,129 +111,92 @@ public Path generate(String id,
         return fullPath;
     }
 
-    public void generate(Path packagePath,
-                         String id,
-                         SemanticVersion version,
-                         String... dependencyList)
-            throws IOException, JAXBException, NoSuchAlgorithmException, NugetFormatException
-    {
+    public void generate(Path packagePath, String id, SemanticVersion version, String... dependencyList)
+            throws IOException, JAXBException, NoSuchAlgorithmException, NugetFormatException {
         generate(packagePath, id, version, new Random().nextInt(DEFAULT_BYTE_SIZE), dependencyList);
     }
 
-    public void generate(Path packagePath,
-                         String id,
-                         SemanticVersion version,
-                         long bytesSize,
-                         String... dependencyList)
-            throws IOException, JAXBException, NoSuchAlgorithmException, NugetFormatException
-    {
+    public void generate(Path packagePath, String id, SemanticVersion version, long bytesSize, String... dependencyList)
+            throws IOException, JAXBException, NoSuchAlgorithmException, NugetFormatException {
         Nuspec nuspec = generateNuspec(id, version, dependencyList);
 
         createArchive(nuspec, packagePath, bytesSize);
 
         generateNuspecFile(nuspec);
     }
 
-    public void createArchive(Nuspec nuspec,
-                              Path packagePath,
-                              long bytesSize)
-            throws IOException,
-                   JAXBException,
-                   NoSuchAlgorithmException
-    {
+    public void createArchive(Nuspec nuspec, Path packagePath, long bytesSize)
+            throws IOException, JAXBException, NoSuchAlgorithmException {
         // Make sure the artifact's parent directory exists before writing the model.
         Files.createDirectories(packagePath.getParent());
 
-        try (OutputStream fileOutputStream = Files.newOutputStream(packagePath))
-        {
-            LayoutOutputStream layoutOutputStream = new LayoutOutputStream(fileOutputStream);
-            layoutOutputStream.addAlgorithm(MessageDigestAlgorithms.SHA_512);
-            layoutOutputStream.setDigestStringifier(this::toBase64);
+        try (OutputStream fileOutputStream = Files.newOutputStream(packagePath)) {
+            try (LayoutOutputStream layoutOutputStream = new LayoutOutputStream(fileOutputStream)) {
+                layoutOutputStream.addAlgorithm(MessageDigestAlgorithms.SHA_512);
+                layoutOutputStream.setDigestStringifier(this::toBase64);
 
-            try (ZipOutputStream zos = new ZipOutputStream(layoutOutputStream))
-            {
-                addNugetNuspecFile(nuspec, zos);
-                TestFileUtils.generateFile(zos, bytesSize, "file-with-given-size");
+                try (ZipOutputStream zos = new ZipOutputStream(layoutOutputStream)) {
```

Comment:
Why are there so many levels of nested `try`/`catch` blocks here? What am I missing? (Also, similarly to my above remarks, braces on new lines and so on).